index.js

```diff
@@ -1,9 +1,9 @@
 function add (a, b) {
-  return a + b
+  return a + b;
 }
 
 function multiply (a, b) {
-  return a * b
+  return a * b;
 }
 
 function subtract (a, b) {
```

Comment:
Add a semicolon in this function as well!

Reply:
Added!